packages/server/postgres/migrations/1644936872423_addTeamPromptResponseTable.ts

```diff
@@ -0,0 +1,72 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+    DO $$
+    BEGIN
+      IF NOT EXISTS (SELECT 1 FROM pg_type WHERE typname = 'Reactji') THEN
+        CREATE TYPE "Reactji" AS (id text, userId text);
+      END IF;
+
+      ALTER TYPE "DiscussionTopicTypeEnum" ADD VALUE 'teamPromptResponse';
+
+      ALTER TYPE "MeetingTypeEnum" ADD VALUE 'teamPrompt';
+
+      CREATE TABLE IF NOT EXISTS "TeamPromptResponse" (
+        "id" VARCHAR(50) PRIMARY KEY,
```

Comment:
+1 curious why no `"id" INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY`
Was it no good for the polls table?

Reply:
I focused too much on enums and composite types and actually didn't think too much about this, very much my bad, it should be exactly like in polls!